dspace-api/src/main/java/org/dspace/discovery/indexobject/IndexFactoryImpl.java

```diff
@@ -73,7 +73,12 @@ public SolrInputDocument buildDocument(Context context, T indexableObject) throw
     @Override
     public void writeDocument(Context context, T indexableObject, SolrInputDocument solrInputDocument)
             throws SQLException, IOException, SolrServerException {
-        writeDocument(solrInputDocument, null);
+        try {
+            writeDocument(solrInputDocument, null);
+        } catch (Exception e) {
+            log.error("Error occurred while writing SOLR document for {} object {} {}",
```

Comment:
One too many sets of braces?  If we do want a trace, I think we do not want the Exception to be consumed by the message formatter.

Reply:
I think this is the only way, there doesn't seem to be a method that takes two objects and the exception. (i would however still need to add `e.getStacktrace()` for it to be actually displayed). But i tested this local and it does work as expected.

Reply:
Your point is correct, i have now removed and assured this works locally